### ISSUE TITLE
Issue while running Medium notebook

### ISSUE THREAD

**Issue by gabrielnetz** [gaphex/bert_experimental#18]
Hi, first of all, thanks a lot for the effort in the Medium articles on BERT, i found them super helpful for my research

I'm trying to run [this](https://towardsdatascience.com/building-a-search-engine-with-bert-and-tensorflow-c6fdc0186c8a) notebook, but it does not seem to be working anymore on colab, due to changes on the standardized TF and Python versions (It defaults to 2.1 and  3.7 nowadays). Would you mind giving me the original requirements which were used so i can reproduce the experiment?

**Comment by Ab1992ao** [comment]
Hello, my friend. We so sorry, but this code working with tf.1x versions.

пн, 7 мар. 2022 г., 8:19 PM Gabriel Netz ***@***.***>:

**Comment by gabrielnetz** [comment]
Yeah, i saw its all running on tf.1x, but even after changing the tensor flow version on colab, it still doesn't run properly, so i imagine there must be something else breaking it. Tried changing python to 3.6 (last colab supported version), but no luck :(.

It gives an error when running the inference graph, any clue on how i should go about running that code?
![image](https://user-images.githubusercontent.com/51419386/157088280-e19a0ea4-d1b2-488a-916d-429c1d33ceb0.png)
